ecology.yaml

```diff
@@ -0,0 +1,515 @@
+
+  - name: suite_datamash
+    owner: iuc
+    tool_panel_section_label: Join, Subtract and Group
+
+
+  - name: subtract
+    owner: devteam
+    tool_panel_section_label: Join, Subtract and Group
+
+
+  - name: numeric_clustering
+    owner: bgruening
+    tool_panel_section_label: Statistics
+  - name: t_test_two_samples
+    owner: devteam
+    tool_panel_section_label: Statistics
+  - name: seq_composition
+    owner: peterjc
+    tool_panel_section_label: Statistics
+  - name: plot_from_lda
+    owner: devteam
+    tool_panel_section_label: Statistics
+  - name: mine
+    owner: devteam
+    tool_panel_section_label: Statistics
+  - name: lda_analysis
+    owner: devteam
+    tool_panel_section_label: Statistics
+  - name: generate_pc_lda_matrix
+    owner: devteam
+    tool_panel_section_label: Statistics
+  - name: count_gff_features
+    owner: devteam
+    tool_panel_section_label: Statistics
+  - name: correlation
+    owner: devteam
+    tool_panel_section_label: Statistics
+
+
+  - name: t_test_two_samples
+    owner: devteam
+    tool_panel_section_label: Statistics
+
+  - name: plot_from_lda
+    owner: devteam
+    tool_panel_section_label: Statistics
+
+  - name: mine
+    owner: devteam
+    tool_panel_section_label: Statistics
+
+  - name: lda_analysis
+    owner: devteam
+    tool_panel_section_label: Statistics
+
+  - name: generate_pc_lda_matrix
+    owner: devteam
+    tool_panel_section_label: Statistics
+
+  - name: count_gff_features
+    owner: devteam
+    tool_panel_section_label: Statistics
+
+  - name: correlation
+    owner: devteam
+    tool_panel_section_label: Statistics
+
+
+
+  - name: venn_list
+    owner: peterjc
+    tool_panel_section_label: Graph/Display Data
+  - name: scatterplot
+    owner: devteam
+    tool_panel_section_label: Graph/Display Data
+  - name: histogram
+    owner: devteam
+    tool_panel_section_label: Graph/Display Data
+  - name: gmaj
+    owner: devteam
+    tool_panel_section_label: Graph/Display Data
+  - name: xy_plot
+    owner: devteam
+    tool_panel_section_label: Graph/Display Data
+  - name: draw_stacked_barplots
+    owner: devteam
+    tool_panel_section_label: Graph / Display Data
+
+  - name: ggplot2_pca
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: pygenometracks
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: ggplot2_heatmap
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: ggplot2_violin
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: ggplot2_heatmap2
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: ggplot2_point
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: ggplot2_histogram
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: tsne
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: best_regression_subsets
```

Comment:
We need to check all devteam tools if they are either already installed or if we need to condafy them.

Reply:
They probably are. If you say all of these are already in. Thats fine. I was just worried by the duplication and one of the first tool is a very old one from devteam that I try to migrate to conda in https://github.com/galaxyproject/tools-devteam/pull/510

Reply:
ok, thanks!